code/modules/mod/mod_theme.dm

```diff
@@ -166,13 +165,12 @@
 	armor = list(MELEE = 20, BULLET = 5, LASER = 5, ENERGY = 5, BOMB = 30, RAD = 150, FIRE = INFINITY, ACID = 150) //Bomb armor bumped up a bit, as the modsuit describes it with blast-dampening
 
 /datum/mod_theme/atmospheric
-	name = "atmospheric"
-	desc = "An atmospheric-resistant suit by Cybersun Industries, offering extreme heat resistance compared to the engineer suit."
-	extended_desc = "A modified version of the Cybersun Industries industrial model. This one has been \
-		augmented with the latest in heat-resistant alloys, paired with a series of advanced heatsinks. \
-		Additionally, the materials used to construct this suit have rendered it extremely hardy against \
-		corrosive gasses and liquids, useful in the world of pipes. \
-		However, the capacity for modification remains the same as civilian-grade suits."
+	name = "'Canary' atmospheric"
+	desc = "A reinforced atmospherics modsuit meant for extreme environments. Completely fireproof, but somewhat lacking in modification potential and power efficiency."
+	extended_desc = "Developed from the popular 'Spark' chassis, the CS-16 'Canary' utility modsuit is specialised for use by atmospherics specialists. The outer carapace is made of highly insulating thermally-reflective composites \
+		underlain with multiple layers of insulating fiberglass, which, in combination with a high-powered thermal regulation system, provides the user complete protection from even full engulfment inside a raging plasma fire. \
+		The helmet contains integrated filtration systems that protect the user from sudden releases of gaseous substances, as sometimes encountered during scrubber system backflows - a feature not present in the Engineering MOD. \
```

Comment:
This feels like it's included somewhat awkwardly, I think it could probably be worded better in ways that don't specifically call out scrubber backflows?